cpp/log/cert.cc

```diff
@@ -723,6 +1064,7 @@ Cert::Status TbsCertificate::ExtensionIndex(int extension_nid,
   return Cert::TRUE;
 }
 
+
 CertChain::CertChain(const string& pem_string) {
   // A read-only BIO.
   BIO* bio_in = BIO_new_mem_buf(const_cast<char*>(pem_string.data()),
```

Comment:
This was there before. I just got rid of std:: via global replace, same for next comment.